tryalgo/interval_cover.py

```diff
@@ -11,16 +11,16 @@ def _readarray(f): return map(f, stdin.readline().split())
 
 
 def _solve(iles, rayon):
-    I = []
+    II = []
```

Comment:
Why is this better?

Reply:
pylint was complaining (C0103) for one-character naming but we can leave one-character and bypass this rule